content/en/drone-peripheral-assembly/gimbal-assembly/_index.md

```diff
@@ -0,0 +1,236 @@
+
+![Tilt servo attached to gimbal pan part](gimbal_pan_tilt_servo.jpg)
+
+## Gimbal Tilt
+
+Next we will be assembling the tilt mechanism.
+This is where all of our sensors will be mounted.
+
+### Thermal Camera Mounting
+
+The thermal camera will be mounted in the center of our gimbal tilt part.
+Use 4 screws from the peripheral bag to attach the thermal camera to the drone.
+
+![Thermal Camera Mounting Overview](gimbal_thermal_overview.jpg)
+
+After screwing in, it should look like the photo below.
+
+{{% alert title="Note" color="note" %}}
+Make note of the thermal camera's orientation. This will be important for properly viewing the thermal image in the AVR GUI.
+{{% /alert %}}
+
+![Mounted Thermal Camera](gimbal_thermal.jpg)
+
+We will wire and test this in following sections.
+
+### FPV mounting
+
+The FPV camera is mounted on the left side of the part (as shown in the picture).
+
+![FPV Mounting Overview](gimbal_fpv_overview.jpg)
+
+Cut a 3M double sided tape to the size of the base of the protruding area of the tilt part and place.
+
+![FPV Double Sided Tape Placement](gimbal_fpv1.jpg)
+
+Slide the camera into the slot and ensure that the camera is secured.
+You should be able turn the part upside down without the camera falling out.
+
+![FPV Mounted in Tilt Assembly](gimbal_fpv2.jpg)
+
+We will wire and test this in following sections.
+
+### Mount Servo Horn
+
+Next place your servo horn into the the recess on the tilt part.
+We find that using a small amount of super glue is helpful in holding the servo in place before we attach it.
+
+![Tilt Servo Horn Mounting](gimbal_tilt_servohorn_mount.jpg)
+
+The only remaining part of the gimbal assembly components is the laser,
+but we will come back to that after fully assembling the gimbal on the drone.
+
+## Gimbal Assembly
+
+Now that all of the assembly 3D parts have been assembled we can start mounting the gimbal to the drone.
+
+{{% alert title="Note" color="note" %}}
+Before performing this section, ensure you have connected to your drone with
+[the GUI]({{< relref "../../Software/GUI" >}})
+as we will need to use it in order to calibrate the Gimbal.
+{{% /alert %}}
+
+First, mount the Gimbal base to the drone using 10mm 3M screws included in your kit.
+Use the 1st and 3rd slotted holes from the front of the drone to mount the gimbal base.
+
+![Gimbal Base Mounted](gimbal_base_mount1.jpg)
+
+![Lock Screws](gimbal_base_mount2.jpg)
+
+### Pan and Tilt Calibration
+
+Please watch the video below to understand calibrating your gimbal's pan and tilt range of motion.
+
+{{< youtube _mtDJYaxGYs >}}
+
+{{% alert title="Note" color="note" %}}
+If you do not have the "invert joystick" option in your GUI be sure you have the
+[latest stable release](https://github.com/bellflight/AVR-2022/releases/tag/stable)
+of the GUI.
+{{% /alert %}}
+
+Code to connect to GUI:
```

Comment:
This isn't code, it's commands, and there should be more explanation on where/why to run this.

Reply:
Changed it to command not code. Good catch thank you. 

The where an when is defined in the video pretty clearly. I included the commands cause I thought it would be a nice ease of use thing.